infrastructure/stacks/capacity_management/dynamodb.tf

```diff
@@ -0,0 +1,67 @@
+module "dynamodb_organisations_table" {
+  source = "../../modules/dynamodb"
+
+  table_name          = var.organisations_table_name
+  hash_key            = "id"
+  autoscaling_enabled = true
```

Comment:
We have hardcoded values here for eg autoscaling method but we also define variables (below) with default vales. Why not plug those variables in now ?